### PR TITLE
Add login page

### LINKED ISSUE #2: Add login page

### PR DESCRIPTION
**Pull request by optiman917**

### CODE REVIEW

File: nuxt.config.js [hunk 1]
```diff
@@ -57,9 +57,7 @@ export default {
    ** Axios module configuration
    ** See https://axios.nuxtjs.org/options
    */
-  axios: {
-    baseURL: process.env.BASE_URL || 'http://localhost:3000/api',
```
Comment (east2dd) [inline nuxt.config.js:61]: when will be axios configured?

Comment (east2dd) [reply nuxt.config.js:61]: @liudavid0107 ☝️

File: nuxt.config.js [hunk 2]
```diff
@@ -78,4 +76,7 @@ export default {
     }
   },
   ignore: ['**/*.test.*', '**/*.spec.*'],
+  publicRuntimeConfig: {
+    baseURL: process.env.BASE_URL || 'http://localhost:8080/api',
+  },
```
Comment (optiman917) [inline nuxt.config.js]: From Nuxt v2.13 they use runtime config.

File: nuxt.config.js (hunk 1 above)
Comment (optiman917) [reply nuxt.config.js:61]: From Nuxt v2.13, they use runtime config.

Comment (east2dd) [reply nuxt.config.js:61]: so, axios will be configured automatically from runtime config?

Comment (optiman917) [reply nuxt.config.js:61]: Yes, I think so.

File: nuxt.config.js (hunk 2 above)
Comment (east2dd) [reply nuxt.config.js]: I know its runtime config, but please make sure axios is configured with this variable

Comment (optiman917) [reply nuxt.config.js]: OK

**Comment by optiman917** [comment]
Done. Please review my PR. @xyingsoft

File: nuxt.config.js [hunk 3]
```diff
@@ -78,4 +78,7 @@ export default {
     }
   },
   ignore: ['**/*.test.*', '**/*.spec.*'],
+  publicRuntimeConfig: {
+    baseURL: process.env.BASE_URL || 'http://localhost:8080/api',
```
Comment (east2dd) [inline nuxt.config.js]: where will we use this variable?

Comment (optiman917) [reply nuxt.config.js]: It is baseUrl for api server. But I configured this URL in axios option. So I will delete it.

Comment (east2dd) [reply nuxt.config.js]: Ok 👍

**Review by east2dd — APPROVED** [review]

**Pull request merged**